ansible/roles/gcp-get-token/tasks/main.yml

```diff
@@ -0,0 +1,73 @@
+---
+- name: Load GCP credentials into string
+  set_fact:
+    gcp_creds: "{{ gcp_credentials | from_json }}"
```

Comment:
Why is `gcp_credentials` passed as a json blob? Why not just have it as normal structured data?

Reply:
because this comes from a file that google provides.  its used by the collection module and has to be a blob